lua/entities/gmod_wire_expression2/core/entity.lua

```diff
@@ -773,6 +773,15 @@ e2function void entity:podStripWeapons()
 	end
 end
 
+e2function void entity:podSetName(string name)
+	if not IsValid(this) or not this:IsVehicle() or not this.VehicleTable or not this.VehicleTable.Name then return self:throw("Invalid vehicle!", nil) end
+	if not isOwner(self, this) then return self:throw("You do not own this entity!", nil) end
+	if #name > 200 then
+			name = name:sub(1,200)
+	end
```

Comment:
This can be simplified to just this

```suggestion
	name = name:sub(1,200)
```